src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -444,15 +444,15 @@ const NotebookEditorFrame = ({ mode, notebookName, workspace: { workspace: { nam
       }),
       h(JupyterFrameManager, {
         frameRef,
-        onClose: () => Nav.goToPath('workspace-notebooks', { namespace, name }),
-        details: { notebookName, name, namespace }
+        onClose: () => Nav.goToPath('workspace-notebooks', { googleProject, name }),
```

Comment:
Also think `Nav.goToPath(…)` should still use `namespace`. The navPaths are still based on namespace: https://github.com/DataBiosphere/terra-ui/blob/2b2d8ca1640ca4293d8a48c443fdc74a36ea15bd/src/pages/workspaces/workspace/Notebooks.js#L486